app/build.gradle.kts

```
@@ -48,6 +48,7 @@ dependencies {
     implementation("androidx.appcompat:appcompat:1.3.1")
     implementation("com.google.android.material:material:1.4.0")
     implementation("androidx.constraintlayout:constraintlayout:2.1.0")
+    implementation(project(mapOf("path" to ":domain")))
```

```suggestion
    implementation(project(":domain"))
```
이렇게 작성하셔도 됩니다 😄 (이하 관련 내용 동일)